sesman/libsesman/sesman_config.c

```diff
@@ -614,39 +638,16 @@ config_dump(struct config_sesman *config)
 
     /* Security configuration */
     g_writeln("Security configuration:");
-    g_writeln("    AllowRootLogin:            %d", sc->allow_root);
+    g_writeln("    AllowRootLogin:  esm         %d", sc->allow_root);
```

Comment:
What does esm mean？

Reply:
That looks like finger trouble to me - I'll remove it. Thanks!